src/portable/portable.c

```diff
@@ -1538,6 +1534,11 @@ int portable_attach(
          * operation otherwise. */
         (void) install_image_and_extensions_symlinks(image, extension_images, flags, changes, n_changes);
 
+        log_info("Successfully attached '%s%s%s'",
+                 image->path,
+                 isempty(extensions_joined) ? "" : "' and its extension(s) '",
+                 strempty(extensions_joined));
```

Comment:
hmm, any chance you can add the extra work to make this structured, i.e. use log_struct(), make up a new message id for this, and add a brief catalog entry for this?

this looks like a fundamental message that should be recognizable.

Reply:
Yeah I had thought about that as well, done now.